tracer/test/test-applications/Samples.Shared/Samples.Shared.shproj

```diff
@@ -1,11 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-    <PropertyGroup Label="Globals">
-        <ProjectGuid>{5A806F4B-39E7-4F38-B36F-F5CFC4F8760A}</ProjectGuid>
-    </PropertyGroup>
-    <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
-    <Import Project="$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)\CodeSharing\Microsoft.CodeSharing.Common.Default.props" />
-    <Import Project="$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)\CodeSharing\Microsoft.CodeSharing.Common.props" />
-    <Import Project="Samples.Shared.projitems" Label="Shared" />
-    <Import Project="$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)\CodeSharing\Microsoft.CodeSharing.CSharp.targets" />
-</Project>
+  <PropertyGroup Label="Globals">
+    <ProjectGuid>{5A806F4B-39E7-4F38-B36F-F5CFC4F8760A}</ProjectGuid>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="TraceAttribute.cs" />
+  </ItemGroup>
```

Comment:
I don't think the changes in this file are required? 🤔

Reply:
Uhm, VS did it automatically, but you are right, it's not needed. Reverting